app/src/components/Courses/PersonalInsights.tsx

```diff
@@ -19,52 +25,89 @@ type PersonalInsightsProps = {
 const PersonalInsights = (props: PersonalInsightsProps) => {
   const courses = props.courses ?? [];
 
+  // Gets the total number of subscribers from all courses written by the content creator
+  function getAllSubscribers() {
```

Comment:
Hvorfor er getAllSubscribers, getNumberOfCourses, getAverageRating, getCourseCountWithStatus ikke funktioner i userinfo

Reply:
Moved them into a courses.ts helper. Doesn't really make sense to have them together with userInfo 👍

Reply:
Sounds good